com.ibm.streamsx.topology/opt/python/packages/streamsx/spl/runtime.py

```diff
@@ -41,23 +41,26 @@ def _wf():
 # may be sparse, values not set by the dictionary
 # (etc.) are set to None in the Python tuple.
 
+from streamsx.spl.types import valueNull as types
```

Comment:
Not sure what is going on here, would seem it's trying to call `valueNull.valueNull()` ???